docs/semgrep-ci/overview.md

```diff
@@ -253,32 +253,3 @@ Some rules point out hotspots that require careful review but are not certain to
 You can set this up through the [Rule Board](/semgrep-app/rule-board).
 
 <MoreHelp />
-
-<!-- # Technical details
-
-## Behavior
-
-Semgrep CI uses environment variables
```

Comment:
Was explaining the diff aware scans moved somewhere else?

Reply:
@brendongo not sure, but this section has been commented out with `<!-- -->` for like a year.

Reply:
Oh I see. Sg.